app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def update_posts_comments_counter
+    post.increment!(:comments_counter)
+    post.save
+  end
```

Comment:
- [ ] **[OPTIONAL]** There is no need to call `.save` here. The first line should be enough. We can also add a callback for this method and make it private so that when a like is created it is called automatically.

  ```
  after_save :update_posts_comments_counter

  ...
  
  private

  def update_posts_comments_counter
    post.increment!(:comments_counter)
  end
  ```

Reply:
Thanks @TedLivist i will make the changes